styles.css

```diff
@@ -0,0 +1,78 @@
+:root {
+  --black-color: #1a2236;
+}
+
+* {
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+}
+
+body {
+  background: url("Images/image_geometry_1.png");
+  background-repeat: no-repeat;
+  position: relative;
+}
+
+.mobile-nav {
+  display: flex;
+  justify-content: flex-end;
+  padding-right: 16px;
+  padding-top: 19px;
+}
+
+.name-tag {
```

Comment:
[OPTIONAL]

- [ ] Good job implementing the changes by the previous reviewer 👍🏼. To align the title to the left, kindly remove the `align-items: center` on the` .body-display` class as this will center all the elements in the section after which you add margin left and right or padding to the section as specified in the design. Then your h1 element will perfectly match the Figma design 😄.

Reply:
Thanks a lot, @vickymarz I appreciate it.